pkg/auth/prometheus/metrics.go

```diff
@@ -0,0 +1,36 @@
+package prometheus
```

Comment:
Call this package "metrics", we have no plans to support an alternate metrics sysem